app/models/user.rb

```diff
@@ -2,9 +2,68 @@ class User < ApplicationRecord
   has_many :posts, dependent: :destroy
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+  has_many :friendships, foreign_key: "creator_id"
```

Comment:
Hi Maya, when creating our initial friendships table, we decided to create columns for creator and recipient, so that it was clear who created a friendship, and who received a friend request. This maps over to our associations with :friendships and :inverse_friendships. Hope that sound alright. We made the other changes, thanks!

Reply:
I meant that having a foreign key that is not unique is not recommended. `creator_id` is not unique, in fact, you do not need this foreign key in your association

Reply:
Think about it, a user may send friend requests to people and when they accept they are the creator in multiple friendships. It is not unique in the original table so it should not be a foreign key. Does this make sense to you? Or is my explanation poor? 😅

Reply:
That being said, I understand what you are saying, so this is fine! 😄

Reply:
This is ok